app/build.gradle

```diff
@@ -171,6 +171,9 @@ dependencies {
     implementation "org.jetbrains.anko:anko-commons:$anko_version"
     implementation "org.jetbrains.anko:anko-design:$anko_version"
 
+    //Mapbox java sdk
+    implementation 'com.mapbox.mapboxsdk:mapbox-sdk-services:4.5.0'
```

Comment:
Why the whole SDK and not just places search?

Reply:
all the services are tightly bound .  This is the services part only (Geocoding , Directions etc)

```
implementation 'com.mapbox.mapboxsdk:mapbox-sdk-core:4.6.0'
implementation 'com.mapbox.mapboxsdk:mapbox-sdk-geojson:4.6.0'
implementation 'com.mapbox.mapboxsdk:mapbox-sdk-services:4.6.0'
implementation 'com.mapbox.mapboxsdk:mapbox-sdk-turf:4.6.0'
```

These are available options only 

As we need Geocoding service , it would be provided by this services part only. Thats why i added

Reply:
This is the size of PlayStoreRelease Apk 5.3MB

![image](https://user-images.githubusercontent.com/24780524/56075290-beb1f500-5ddd-11e9-8377-65a9b0b59c51.png)